hw/dv/tools/dvsim/sim.mk

```diff
@@ -115,9 +115,24 @@ cov_unr_build: gen_sv_flist
 	@echo "[make]: cov_unr_build"
 	cd ${sv_flist_gen_dir} && ${cov_unr_build_cmd} ${cov_unr_build_opts}
 
-cov_unr: cov_unr_build
+#cov_unr: cov_unr_build
+#	@echo "[make]: cov_unr"
+#	cd ${sv_flist_gen_dir} && ${cov_unr_run_cmd} ${cov_unr_run_opts}
+
+cov_unr_run: 
+ifeq ("$(wildcard $(cov_merge_db_dir))", "")
+	@echo "NO COVERAGE DATABASE DEFINED"
+else
 	@echo "[make]: cov_unr"
-	cd ${sv_flist_gen_dir} && ${cov_unr_run_cmd} ${cov_unr_run_opts}
+	mkdir -p ${cov_unr_dir}
+	cd ${cov_unr_dir}; ${cov_unr_run_cmd} ${cov_unr_run_opts}
+endif
+
+cov_unr_merge:
+	 ${cov_merge_cmd} -init ${cov_unr_dir}/jgproject/sessionLogs/session_0/unr_imc_coverage_merge.cmd 
```

Comment:
this is only needed for xcelium. can we move this to `xcelium.hjson`

Reply:
not sure what you are saying - the Make still have to call this! so moving it won't really make a difference, or maybe I am just misunderstanding your point!

Reply:
If we run VCS unr, this makefile will be called but this target `cov_unr_merge` is only applicable for Xcelium.

If you move it to xcelium.hjson. Then it won't affect VCS unr. Does it make sense?